src/main/java/com/derongan/minecraft/mineinabyss/Relic/Behaviour/Behaviours/CampfireRelicBehaviour.java

```diff
@@ -0,0 +1,57 @@
+package com.derongan.minecraft.mineinabyss.Relic.Behaviour.Behaviours;
+
+import com.derongan.minecraft.mineinabyss.Relic.Behaviour.UseRelicBehaviour;
+import com.derongan.minecraft.mineinabyss.Relic.Relics.RelicType;
+import com.derongan.minecraft.mineinabyss.Relic.Relics.SlightlyOffzRelicType;
+import org.bukkit.Material;
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockFace;
+import org.bukkit.entity.*;
+import org.bukkit.event.block.Action;
+import org.bukkit.event.player.PlayerInteractEvent;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.EulerAngle;
+
+public class CampfireRelicBehaviour implements UseRelicBehaviour {
+    @Override
+    public void onUse(PlayerInteractEvent event) {
+        if (event.getAction().equals(Action.RIGHT_CLICK_BLOCK) && event.getBlockFace() == BlockFace.UP) {
+            event.setCancelled(true);
+
+            Player player = event.getPlayer();
+
+            Block target = event.getClickedBlock();
+
+            Material ma = target.getType();
+
+            if(ma.equals(Material.GRASS) || ma.equals(Material.STONE)) {
```

Comment:
Id expand this to a list.

List<Material> allowedPlacements = Arrays.asList({Material.GRASS, Material.STONE, Material.DIRT....});
...
if(allowedPlacements.contains(ma))
...

Also not that because Material.GRASS is an enum, you can check using ==